web/frontend/views.py

```diff
@@ -1,52 +1,65 @@
+import json
+import logging
+
 import stripe
 from accounts.models import CustomUser, Team
 from django.conf import settings
+from django.contrib import messages
 from django.http import JsonResponse
 from django.shortcuts import redirect, render
 from django.urls import reverse
 from django.views.decorators.csrf import csrf_exempt
 from django.views.decorators.http import require_POST
 
+logger = logging.getLogger(__name__)
+
 
 def home(request):
     return render(request, "home.html")
 
 
 def payment_success(request):
-    return render(request, "payment_success.html")
+    messages.success(request, "Payment successful!")
+    return redirect("edit_account")
 
 
 def payment_cancel(request):
-    return render(request, "payment_cancel.html")
+    messages.success(request, "Payment cancelled.")
+    return redirect("home")
 
 
 def stripe_checkout(request):
     stripe.api_key = settings.STRIPE_SECRET_KEY
-    success_url = request.build_absolute_uri(reverse("home"))
-    cancel_url = request.build_absolute_uri(reverse("home"))
+    success_url = request.build_absolute_uri(reverse("payment_success"))
+    cancel_url = request.build_absolute_uri(reverse("payment_cancel"))
     email = request.user.email
 
     # Get the subscription type from the query parameter
     subscription_type = request.GET.get(
         "tier", "individual"
     )  # Default to 'individual' if not provided
 
-    price_id = settings.STRIPE_INDIVIDUAL_PRICE_ID
+    price_id_recurrent = settings.STRIPE_INDIVIDUAL_PRICE_ID_RECURRENT
```

Comment:
so now the monthly fees and the usage bills are tracked through different stripe products? Why did we change it from only one?

Reply:
oh I see that makes sense